packages/near-membrane-base/src/membrane.ts

```diff
@@ -4299,15 +4307,20 @@ export function createMembraneMarshall(
                 : (noop as CallableGetLazyPropertyDescriptorStateByTarget),
             // callableGetPropertyValue
             !IS_IN_SHADOW_REALM
-                ? (targetPointer: Pointer, index: PropertyKey) => {
+                ? (targetPointer: Pointer, key: PropertyKey): PointerOrPrimitive => {
                       targetPointer();
                       const target = selectedTarget!;
                       selectedTarget = undefined;
+                      let value: any;
                       try {
-                          return (target as any)[index];
+                          value = (target as any)[key];
                       } catch (error: any) {
                           throw pushErrorAcrossBoundary(error);
                       }
+                      return (typeof value === 'object' && value !== null) ||
```

Comment:
IsObject here?

Reply:
Since this is deep within the near-membrane we inline all helper calls like that.